mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -2476,12 +2477,18 @@ def Vector_TransposeOp :
 }
 
 def Vector_PrintOp :
-  Vector_Op<"print", []>,
+  Vector_Op<"print", [
+    PredOpTrait<
+      "`source` or `punctuation` are not set when printing strings",
```

Comment:
Perhaps I am a bit pedantic, but this reads a bit off :) Perhaps "Strings mixed with `source` and `punctations` are not allowed/supported" ?

Reply:
When this constraint fails it outputs:
```
op failed to verify `source` or `punctuation` are not set when printing strings
```
If I switch it, that does not really work :disappointed: 
```
op failed to verify strings mixed with source and punctations are not allowed/supported
```